packages/sx05re/emulators/flycastsa/package.mk

```diff
@@ -20,7 +20,14 @@ PKG_CMAKE_OPTS_TARGET+="-DUSE_GLES=ON -DUSE_VULKAN=OFF"
 }
 
 makeinstall_target() {
+	mkdir -p $INSTALL/usr/config/flycast
+	mkdir -p $INSTALL/usr/config/flycast/mappings
```

Comment:
Is this needed?

Reply:
I made the mkdirs in dolphin.sh, yes they are needed if you use gamepad auto detection.